src/main/java/com/wavefront/sdk/common/clients/WavefrontClient.java

```diff
@@ -118,7 +118,7 @@
      * @param server A server URL of the form "https://clusterName.wavefront.com" or "http://internal.proxy.com:port"
      * @param token  A valid API token with direct ingestion permissions
      */
-    protected Builder(String server, @Nullable String token) {
+    public Builder(String server, @Nullable String token) {
```

Comment:
Can we just add an overloaded builder that just takes in the server (for proxy)?